static/styles/left_sidebar.css

```diff
@@ -512,6 +521,10 @@ li.expanded_private_message {
     width: 236px;
 }
 
+.topic-list-filter {
+    width: 190px;
+}
```

Comment:
A comment explaining these numbers would be great, though probably better would be to do a prep commit using a `calc` value based on the left sidebar width.  

(This is valuable because we made that width a variable recently).

Reply:
These CSS changes should be part of this commit, as it is part of how filter input is displayed. But I've converted it to set its value using `calc`, and also added comments explaining the `calc` statements.

Reply:
Yeah, this is great.

Longer-term we'll want to use SCSS variables for things like that 36px, but this is good enough for it to be readable.